src/ui-components/BalanceInput.tsx

```diff
@@ -86,17 +86,16 @@ const BalanceInput = ({ className, label = '', helpText = '', onChange, placehol
 		>
 			<InputNumber
 				addonAfter={chainProperties[network]?.tokenSymbol}
-				name='balance'
-				className={`text-sm w-full h-[39px] border-[1px] rounded-l-[4px] mt-0 ${inputClassName} placeholderColor`}
+				name={formItemName}
+				className={`text-sm w-full h-[39px] border-[1px] rounded-l-[4px] rounded-r-[0px] mt-0 ${inputClassName} placeholderColor hover:border-[#E5007A]`}
 				onChange={onBalanceChange}
-				placeholder={`${placeholder} ${chainProperties[network]?.tokenSymbol}`}
```

Comment:
why is the token symbol removed from balance input ?

Reply:
Will check and update

Reply:
it was removed because of the design change, now we are just showing "Add Balance" and no token symbol